recipes/license_server.rb

```diff
@@ -23,17 +23,26 @@
 end
 
 data_bag_item = bag.to_hash
+total_available_license_quota = 0
 
 license_groups = data_bag_item.inject('enterprise' => {}) do |hash, (key, value)|
   unless %w(id chef_type data_bag).include? key
     doc = Nokogiri::XML value
     type = doc.at_xpath('/license/payload/type/text()').to_s
+    quota = doc.at_xpath('/license/payload/quota/text()').to_s.to_i
+    expiration_time = doc.at_xpath('/license/payload/expiration_time/text()').to_s.to_i
+    total_available_license_quota += expiration_time > Time.now.to_i ? quota : 0
```

Comment:
As we talked about... you still need to check while summing quota (because you're allocating pools only of enterprise license)